wiki/Beatmapping/Combo/en.md

```diff
@@ -12,7 +12,7 @@ tags:
 
 # Combo
 
-*Not to be confused with the [score multiplier](/wiki/Score).*
+<!-- line to be added eventually: *Not to be confused with [Score multiplier](/wiki/Game_modifiers/Score_multiplier).* -->
```

Comment:
should the path be /wiki/Gameplay/Score_multiplier instead?

Reply:
I'm not sure; #3809 has it under game modifiers atm